gulpfile.js

```diff
@@ -31,12 +31,7 @@ var paths = {
 
 gulp.task('html', function() {
   return gulp.src('src/index.html')
-    .pipe(data(function(file, callback) {
-      // wrap in {events: … } for template
-      eventData('events', function(err, data) {
-        callback(err, data ? {events: data} : null)
-      })
-    }))
+    .pipe(data(eventData('events/2016/*.md')))
```

Comment:
@AverageMarcus - I was thinking about that; though I reckon it might be useful to specify the year while we're compiling the list of events (@spikeheap mentioned planning the 2017 events before the year is out … ) …